resqpy/organize/OrganizationFeature.py.orig

```diff
@@ -0,0 +1,78 @@
+<<<<<<< HEAD
```

Comment:
There is some merge conflict to be resolved here.

Reply:
hmm... I wonder what this .orig file extension is. @cflynn3 is this from the automated conflict resolution step? Do I just have to delete these files?